src/Apps/Artist/Routes/Overview/Components/ArtistCareerHighlight.tsx

```diff
@@ -11,6 +11,7 @@ interface ArtistCareerHighlightProps {
 export const ArtistCareerHighlight: FC<ArtistCareerHighlightProps> = ({
   insight,
 }) => {
+  if (!insight?.description || !insight?.entities.length) return null
```

Comment:
entities might be null and that's likely the bug, so `!insights?.entities?.length`

Reply:
yes!! 🦺